emails/tests/models_tests.py

```diff
@@ -719,327 +564,292 @@ def test_save_server_storage_false_deletes_data(self):
         assert relay_address.generated_for == ""
         assert relay_address.used_on == ""
 
-    def test_save_server_storage_false_deletes_ALL_data(self):
-        test_desc = "test description"
-        test_generated_for = "secret.com"
-        baker.make(
+    def add_four_relay_addresses(self, user: User | None = None) -> list[RelayAddress]:
+        if user is None:
+            user = self.profile.user
+        return baker.make(
             RelayAddress,
-            user=self.profile.user,
-            description=test_desc,
-            generated_for=test_generated_for,
+            user=user,
+            description=self.TEST_DESCRIPTION,
+            generated_for=self.TEST_GENERATED_FOR,
+            used_on=self.TEST_USED_ON,
             _quantity=4,
```

Comment:
praise: TIL `baker.make(_quantity=)` (I know it was in there before but I didn't notice it!)